mungegithub/mungers/matchers/comment/command_test.go

```diff
@@ -45,6 +45,18 @@ func TestParseCommand(t *testing.T) {
 			comment:         "/COMMAND Valid command",
 		},
 		{
+			expectedCommand: &Command{Name: "COMMAND", Arguments: "Multiple Lines"},
```

Comment:
Do we want to support this syntax of the command not being at the beginning of a comment?